jobbergate-cli/tests/subapps/job_scripts/test_app.py

```diff
@@ -16,8 +19,11 @@
     JOB_SUBMISSION_HIDDEN_FIELDS,
     create,
     delete,
+    download_files,
     get_one,
+    handle_abort,
```

Comment:
This import is unised

Reply:
solved in [171c3db](https://github.com/omnivector-solutions/jobbergate/pull/198/commits/171c3db88b03ec2dad7a85c542f5fe99692c8eb3)